packages/admin-ui/src/form-control/form-control-label.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import type { ReactNode } from 'react'
+
+import { useFormControlContext } from './context'
+import { useMessageFormatter } from '../i18n'
+
+import { messages } from './form-control.i18n'
+import { createComponent, useElement } from '..'
+
+export const FormControlLabel = createComponent<
+  'label',
+  FormControlLabelOptions
+>((props) => {
+  const { children, ...htmlProps } = props
+
+  const { optional } = useFormControlContext()
```

Comment:
Just a thought, we could use required instead of optional. This would reuse the input behavior and maybe help integrate with form libraries.

Reply:
I had this discussion with Davis about optional and required, based on what we have in the vtex application he thought it would make more flag optional fields instead of mandatory ones. I think it would be necessary to check how it would the validation of mandatory fields 🤔